bedrock/mozorg/templates/mozorg/mission.html

```diff
@@ -73,7 +73,7 @@ <h1 class="title-shadow-box">{{ _('We’re building a better Internet') }}</h1>
       <p>{{ _('Topics include support, products, and technologies') }}</p>
     </li>
     <li>
-      <h4><a href="/about/governance.html">{{ _('Governance') }}</a></h4>
+      <h4><a href="/about/governance/">{{ _('Governance') }}</a></h4>
```

Comment:
yeah the links work fine as the are, but it's still good to use the URL helper now these pages have been ported to bedrock :+1: